### ISSUE TITLE
New vision/eye phenotype MP term c-wave

### ISSUE THREAD

**Issue by abby-jonesKOMP** [obophenotype/mammalian-phenotype-ontology#3343]
>If this is a request for a new term, please provide us with the following information.

>1. Preferred term label: abnormal c-wave amplitude

>2. Synonyms: n/a

>3. Textual definition: anomaly in the size (height or maximum displacement) of the c-wave as measured in the electroretinogram

>4. Any additional notes to clarify the meaning or use of the term. Please provide a PubMed ID or URL(s) for the source of this information.

>5. Suggested parent term (Please look in the hierarchy of the MP using the [MP Browser](http://www.informatics.jax.org/vocab/mp_ontology) or other browsers such as [OLS](http://www.ebi.ac.uk/ols/ontologies/mp) and find the place in the MP where the new term best fits)

abnormal c wave shape (MP:0012040)

>6. If you would like a nanoattribution, please indicate your ORCID id or provide a label such as MGI:sbello (organization, first initial-last name)

**Comment by sbello** [comment]
added the requested term plus increased and decreased children, will add MP IDs here once this goes public
